### PR TITLE
Added 3 new questions to the microsoft excel

### PR DESCRIPTION
**Pull request by swordwielder**
added 3 new questions to microsoft excel.

### CODE REVIEW

**Comment by swordwielder** [comment]
some additional questions to MS powerpoint.

File: microsoft-excel/microsoft-excel-quiz.md [hunk 1]
```diff
@@ -354,3 +354,24 @@
...
 - [ ] Copy and Paste
+
+#### Q54. You want to add a column to the PivotTable below that shows a 5% bonus for each sales rep.  That data does not exists in the original data table.  How can you do this without adding more data to the table?
+![image](https://user-images.githubusercontent.com/8637045/113378635-7cc14700-9345-11eb-95fd-0f751af49317.png)
+- [ ] Add a new PivotTable field.
+- [ ] Add a calculated item
+- [ ] Add a new Summarize Value By field.
+- [x] Add a calculated field.
+
+#### Q54. You need to determine the commission earned by each Sales rep, based on the Sales amount in B3:B50 and the Commission rate specified in cell A1.  You want to enter a formula in C3 and copy it down to C50.  Which formula should you use?
```
Comment (Ebazhanov) [inline microsoft-excel/microsoft-excel-quiz.md:365]:
```suggestion
#### Q55. You need to determine the commission earned by each Sales rep, based on the Sales amount in B3:B50 and the Commission rate specified in cell A1.  You want to enter a formula in C3 and copy it down to C50.  Which formula should you use?
```

File: microsoft-excel/microsoft-excel-quiz.md [hunk 2]
```diff
@@ -354,3 +354,24 @@
...
+![image](https://user-images.githubusercontent.com/8637045/113378769-cdd13b00-9345-11eb-81f4-b9567d53f26b.png)
+- [ ] =A1*$B3
+- [ ] =A1*B3
+- [x] =$A$1*B3
+- [ ] =$A1*B3
+
+#### Q55. The NOW() function returns the current date and time as 43740.665218.  Which part of this value indicates the time?
```
Comment (Ebazhanov) [inline microsoft-excel/microsoft-excel-quiz.md:372]:
```suggestion
#### Q56. The NOW() function returns the current date and time as 43740.665218.  Which part of this value indicates the time?
```

**Review by Ebazhanov — CHANGES_REQUESTED** [review]
@swordwielder thank you for your pull request I left some minor comments, would be happy to fix them before merge. Thank you in advance!

File: microsoft-excel/microsoft-excel-quiz.md (hunk 2 above)
Comment (swordwielder) [reply microsoft-excel/microsoft-excel-quiz.md:372]: thanks

**Comment by swordwielder** [comment]
Thanks!

**Comment by swordwielder** [comment]
Cool

**Comment by Ebazhanov** [comment]
Wait you can push your changes directly to this branch, or what is your plan?

**Comment by swordwielder** [comment]
I created another pull request.

**Pull request closed without merging**